pythonopensubtitles/opensubtitles.py

```diff
@@ -6,6 +11,36 @@
     from .settings import Settings
 
 
+def decompress_utf_8(data):
+    """
+    Convert a base64-compressed subtitles file with utf-8 encoding
+    back to a string.
+    """
+    decompressed = None
```

Comment:
What about:

```
try:
    return zlib.decompress(base64.b64decode(data), 16 + zlib.MAX_WBITS).decode('utf-8')
except UnicodeError as e
    print(e)
```

I am also thinking that you could move: `zlib.decompress(base64.b64decode(data), 16 + zlib.MAX_WBITS)` to its own function since it's being used in `decompress_latin1` as well.

Reply:
I am also thinking that maybe these functions look better in `utils.py` as you mentioned.

Reply:
You mean to combine the two functions into one and use the encoding as argument (which is the only difference between the two)? Am on that, and also moving it to `utils.py`.

Reply:
Actually, I didn't think about that 😅 But yeah, probably much better. Thanks!